lib/auth_web/controllers/app_controller.ex

```diff
@@ -2,6 +2,8 @@ defmodule AuthWeb.AppController do
   use AuthWeb, :controller
   alias Auth.App
 
+  @auth_url "dwylauth.herokuapp.com"
```

Comment:
we'll actually need to determine this at runtime to avoid people having to update the code. 
e.g. our "Dev" instance https://authdev.herokuapp.com/ should not need to update this line of code. 💭